### ISSUE TITLE
Prepend Section and Paragraph Markers

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/ts-requirements#16]
For each chunk, when it is marked as done, the app should automatically prepend the following:

```
\s5
```

That is our USFM code for identifying the chunk as a chunk.

The app should also add

```
\p
```

after the `\s5` if that is in the source (USX has this as `<para style=\"p\">`).

Those should show up as a separate `commit` from the translator's normal work.

**Comment by RRohrssen** [comment]
Create issues in Android and Desktop

**Comment by da1nerd** [comment]
@jag3773 I do not believe this issue is relevant anymore. 
1. because we are going to use the \z markers
2. because the app is now using the chunks.json to identify chunks
3. because the markers can be added later during compile time. e.g. when exported as USFM.

**Comment by jag3773** [comment]
I agree that we can close this, but for different reasons!  Our publishing scripts are adding the markers when we publish and the files in the repos are already split up.  I can think of some cases where having the chunk marker and/or paragraph marker inserted would be nice, but maybe it's not worth the extra trouble.

Note that part of the reason for this issue was that users cannot insert paragraph markers in tS.  Hence, we end up with a monolithic chunk of text on the output side, with no structural indicators.  This issue could be superseded by an issue to add support for adding paragraph markers (cc @klappy ).
